src/main/webapp/resources/js/components/samples/cartSamplesSlice.ts

```diff
@@ -35,12 +35,13 @@ export const removeCartSampleId = createAction(
  */
 const initialState = (() => {
   return {
-    sampleIds: null,
+    sampleIds: [] as (string | number)[],
```

Comment:
Should these be both?

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f